truss/cli.py

```diff
@@ -190,7 +190,7 @@ def predict(target_directory, request, build_dir, tag, port, use_docker, request
 
     PORT: Local port used to run image
 
-    USE_DOCKER: Flag to run prediction with a docker container
+    RUN_LOCAL: Flag to run prediction with in process rather than on docker container
```

Comment:
is this intentional? I thiought @philipkiely-baseten changed this during the big rename PR?

Reply:
Good catch, @pankajroark this was changed in #174 to be consistent with the client behavior.

Reply:
Oh ok, I misunderstood. This was broken because the click option was still called run-local and was not replaced. Let me change run local to use docker here.